compiler/rustc_lint/src/multiple_supertrait_upcastable.rs

```diff
@@ -46,7 +46,11 @@ impl<'tcx> LateLintPass<'tcx> for MultipleSupertraitUpcastable {
                 .tcx
                 .explicit_super_predicates_of(def_id)
                 .iter_identity_copied()
-                .filter_map(|(pred, _)| pred.as_trait_clause());
+                .filter_map(|(pred, _)| pred.as_trait_clause())
+                .filter(|pred| {
+                    !cx.tcx.is_lang_item(pred.def_id(), hir::LangItem::Sized)
```

Comment:
i'd expect this lint to never trigger for `Sized` :thinking: if `Sized` is a super trait the trait cannot be used as a trait object, can it?

Reply:
Removed `Sized` from this commit.